### DISCUSSION TITLE
Item Stats plugin for Runelite

### DISCUSSION THREAD

**Discussion by Twazor** [runelite/runelite#16946] [Ideas]
Is there a way you can change the statistic panel to only been shown when holding a modifier like Alt or Shift?

The plugin is extremely useful but sometimes during gameplay where I am required to switch multiple pieces of gear quickly, I often find myself lost and not knowing where to click due to the fact that there is an information box above my gear not letting me know where the next click needs to be. So, I lose momentum and fall behind.

I also don't want to be turning the plug in on and off every time I want to compare pieces of equipment because of how clunky the configuration menu is when playing on Resizable mode...

An option to change how the information is displayed by enabling a modifier for the text would be good. After you have compared two pieces of equipment and built your set of gear for the trip you don't really need to see the stats of your DDS every time you go in for the spec.

Thanks for taking the time to read this.

**Comment by EricSciullo** [comment]
There's an open PR to add this feature from 2020: https://github.com/runelite/runelite/pull/11693. Many other games behave this way where you need to hold some key to see the stats and it's much better than the all or nothing we have now. Hopefully someone gets eyes on it.
